<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wg_updata</class>
 <widget class="QWidget" name="wg_updata">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Button_back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_wg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>输入网关</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_wg">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_allserver">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>全部服务器</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_select">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器选择</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>所有服务器</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_servers">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="currentText">
    <string>服务器列表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_updata">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_find">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>280</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>升级记录查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Button_switch_bag">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>选择安装包</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_filename">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_adress">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>180</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入升级地址</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_address">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
